README.md

```diff
@@ -52,9 +52,9 @@ Modules
 -------
 * spock-core -- Core framework. This is the only mandatory module.
 * spock-specs -- Specifications for spock-core, implemented using Spock. Not required for using Spock.
-* spock-spring -- Integration with the [Spring TestContext Framework](http://docs.spring.io/spring/docs/4.1.5.RELEASE/spring-framework-reference/html/testing.html#testcontext-framework).
-* spock-tapestry -- Integration with the [Tapestry 5](http://tapestry.apache.org/tapestry5/) IoC container.
-* spock-guice -- Integration with [Guice](http://code.google.com/p/google-guice/) 2/3.
+* spock-spring -- Integration with the [Spring TestContext Framework](https://docs.spring.io/spring/docs/4.1.5.RELEASE/spring-framework-reference/html/testing.html#testcontext-framework).
+* spock-tapestry -- Integration with the [Tapestry 5](https://tapestry.apache.org/tapestry5/) IoC container.
+* spock-guice -- Integration with [Guice](https://code.google.com/p/google-guice/) 2/3.
```

Comment:
This could be updated to https://github.com/google/guice

Reply:
https://github.com/spockframework/spock/pull/1133/commits/b84a3cbdb92f6d23cbba15955a2b5a74c0fbe695